Remotion/Development/Web/ResourceHosting/DirectoryListingHandler.cs

```diff
@@ -77,25 +78,27 @@ private void RenderBody (HttpContext context, HtmlTextWriter responseWriter)
         
         if (item is ResourceVirtualDirectory)
         {
-          var info = new DirectoryInfo (((ResourceVirtualDirectory) item).PhysicalPath);
```

Comment:
in theory, `null` shouldn't be possible here given we iterate a directory for exisiting children. We can probalby just add an assertion.